lib/model/frames/form.js

```diff
@@ -83,7 +84,10 @@ class Form extends Frame.define(
     const enketoOnceId = (this.def.id === this.currentDefId) ? this.enketoOnceId : null;
     /* eslint-enable indent */
 
-    return Object.assign(Frame.prototype.forApi.call(this), { enketoId, enketoOnceId });
+    // Include deletedAt in response only if it is not null (most likely on resources about soft-deleted forms)
+    const deletedAtField = this.deletedAt ? { deletedAt: this.deletedAt } : null;
```

Comment:
This feels weird but I wasn't sure how else to do this (expose a certain field only in some cases - in particular if it is not null). I also tried it the frame extender way but I ran into ambiguous database column situations and i don't understand the frames stuff well enough to get around it. Thoughts, @issa-tseng?